docs/release/notes.md

```diff
@@ -8,6 +8,8 @@ Release Notes
+| 3.3.29            | 10-Oct-2017  | [Detailed Notes](3.3/release-3-3-29)  | TBA                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
I put the 3.3 releases in a separate table, can you put 3.3.29 in the other one?